post_upload/postupload/release_test.go

```diff
@@ -88,6 +89,7 @@ func TestReleaseToCandidates(t *testing.T) {
 	files := []FileTest{
 		FileTest{"/tmp/src/subdir/file", []string{"pub/firefox/candidates/32-candidates/build23/build-dir/subdir/file"}},
 		FileTest{"/tmp/src/subdir/win32-file", []string{"pub/firefox/candidates/32-candidates/build23/unsigned/build-dir/subdir/win32-file"}},
```

Comment:
We don't use Signed = false any more, it would be good to swap this to true and remove to unsigned/ here.